packages/waku/src/lib/plugins/vite-plugin-deploy-cloudflare.ts

```diff
@@ -231,7 +231,8 @@ export function loadPlatformData(key) {
   switch (key) {
     ${Array.from(keys)
       .map(
-        (k) => `case '${k}': return import('./${DIST_PLATFORM_DATA}/${k}.js');`,
+        (k) =>
+          `case '${k}': return import('./${DIST_PLATFORM_DATA}/${k}.js').then(m => m.default);`,
```

Comment:
This is the core change - other changes in this file were from running prettier.

Also to note - this now mirrors the structure here within the `build.ts` file:

https://github.com/dai-shi/waku/blob/main/packages/waku/src/lib/builder/build.ts#L791